powerbi-docs/admin/service-security-using-microsoft-cloud-app-security-controls.md

```diff
@@ -57,6 +57,20 @@ Sensitivity labels enable you to classify and help protect sensitive content, so
 
 You can read the article on [sensitivity labels in Power BI](service-security-sensitivity-label-overview.md), which goes into detail about the process of using sensitivity labels for Power BI. See below for an [example of a Power BI policy based on sensitivity labels](#example).
 
+## Custom policies to alert on suspicious user activity in Power BI
+
+Cloud App Security activity policy enable administrators to define their own custom rules, to help detect user behavior that deviates from the norm, and even possibly act upon it automatically, if it seems too dangerous. For example:
```

Comment:
either 
A Cloud App Security activity policy enables administrators to define their own custom rules to help detect user behavior ...
or
Cloud App Security activity policies enable administrators to define their own custom rules to help detect user behavior .....